### DISCUSSION TITLE
Prevent caching of requests with invalid responses

### DISCUSSION THREAD

**Discussion by ThomasRutzer** [reduxjs/redux-toolkit#3060] [Q&A]
If I start a query which results in an invalid response (status 4xx for example), I'd like to not cache this query. Is this possible?

**Comment by phryneas** [comment]
That would mean that on every rerender, your component thinks it never made a request and starts a new one. That seems counterproductive to me. What is your use case?

**Reply by ThomasRutzer** [reply]
Ok, I understand that. But in my use case, we are using external services on ther server side and this can sometimes result in flaky responses. So I'd like to give my users a chance to retry. Do you think `refetch` is suitable for that?

**Reply by phryneas** [reply]
Yes, that would be the way to go.
